common/src/main/java/com/wynntils/features/user/players/PlayerGhostArmorTransparencyFeature.java

```diff
@@ -0,0 +1,31 @@
+package com.wynntils.features.user.players;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.FeatureCategory;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.mc.event.PlayerGhostArmorRenderEvent;
+import com.wynntils.wynn.utils.WynnPlayerUtils;
+import com.wynntils.wynn.utils.WynnUtils;
+import net.minecraft.world.entity.player.Player;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@FeatureInfo(category = FeatureCategory.PLAYERS)
+public class PlayerGhostArmorTransparencyFeature extends UserFeature {
```

Comment:
I get that you wanted to do this the "proper" way, but there is no need to make this a separate feature. Just add this is a config option in PlayerghostTransparency feature.